src/ontology/mondo-edit.obo

```diff
@@ -430439,6 +430582,410 @@ property_value: IAO:0000231 "out of scope" xsd:string
+id: MONDO:0035561
+name: sporadic human prion disease
+subset: ordo_group_of_disorders
+synonym: "idiopathic human prion disease" EXACT [Orphanet:576356]
+xref: Orphanet:576356 {source="MONDO:equivalentTo"}
+is_a: MONDO:0018926 {source="Orphanet:576356"} ! human prion disease
+
+[Term]
+id: MONDO:0035562
+name: acquired human prion disease
+subset: ordo_group_of_disorders
+synonym: "infectious human prion disease" EXACT [Orphanet:576360]
+xref: Orphanet:576360 {source="MONDO:equivalentTo"}
+is_a: MONDO:0018926 {source="Orphanet:576360"} ! human prion disease
+
+[Term]
+id: MONDO:0035584
+name: punctate inner choroidopathy
+def: "A rare ophthalmic disorder characterized by typically bilateral, asymmetric, yellowish, punctate chorioretinal lesions of the posterior pole forming a linear branching pattern and progressing to atrophic scars. Subretinal neovascular membranes occur in many cases. Vitritis is always absent. Patients may present with blurred vision, scotoma, floaters, photopsia, and metamorphopsia. Choroidal neovascular membrane formation and subretinal fibrosis are the major causes of visual loss. The condition predominantly occurs in young myopic females." [Orphanet:580951]
+subset: ordo_disorder
+xref: ICD10:H31.0 {source="Orphanet:580951", source="MONDO:mondoSubClassOfSource"}
+xref: Orphanet:580951 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005328 {source="https://orcid.org/0000-0001-5208-3432"} ! eye disorder
+
+[Term]
+id: MONDO:0035592
+name: congenital infiltrating lipomatosis of the face
+subset: ordo_disorder
+synonym: "CIL-F" EXACT [Orphanet:583097]
+synonym: "facial infused lipomatosis" EXACT [Orphanet:583097]
+synonym: "fibroadipose infiltrating lipomatosis" EXACT [Orphanet:583097]
+xref: ICD10:Q87.3 {source="Orphanet:583097", source="MONDO:mondoSubClassOfSource"}
+xref: Orphanet:583097 {source="MONDO:equivalentTo"}
+is_a: MONDO:0006574 {source="https://orcid.org/0000-0001-5208-3432"} ! lipomatosis
+
+[Term]
+id: MONDO:0035614
+name: sporadic fatal insomnia
+def: "A rare sporadic human prion disease characterized by adult onset of progredient neurodegeneration presenting as a combination of psychiatric, sleep, and oculomotor disturbances, with development of progressive cognitive impairment (the predominantly affected cognitive domains being memory, temporal and/or spatial orientation, language, executive functions, and attention), postural instability, and sometimes additional motor abnormalities and autonomic hyperactivity, in the course of the disease. Bilateral thalamic hypometabolism on FDG-PET imaging and positive prion seeding activity in the cerebrospinal fluid are present in many cases. The disease is fatal within typically two to three years." [Orphanet:586130]
+subset: ordo_disorder
+xref: ICD10:A81.9 {source="MONDO:mondoSubClassOfSource", source="Orphanet:586130"}
+xref: Orphanet:586130 {source="MONDO:equivalentTo"}
+is_a: MONDO:0018926 {source="https://orcid.org/0000-0001-5208-3432"} ! human prion disease
```

Comment:
could probably be more specific: sporadic human prion disease
BUT would be a single child

Reply:
I don't see that class in Mondo (sporadic human prion disease)

Reply:
I think 'sporadic human prion disease' is a new term, added here.
id: MONDO:0035561